pkg/manager/manager.go

```diff
@@ -211,3 +213,30 @@ func listenOnResponseChannel(respChan chan events.DeviceResponse) {
 		//TODO handle device connection errors accordingly
 	}
 }
+
+func (m *Manager) computeAndStoreChange(updates map[string]*change.TypedValue,
```

Comment:
It would be enough to just return the configChange.ID - that's the only part of it that's used by the caller anyway